modules/kargoBidAdapter.js

```diff
@@ -4,13 +4,27 @@ import { registerBidder } from '../src/adapters/bidderFactory.js';
 import { getStorageManager } from '../src/storageManager.js';
 import { BANNER, VIDEO } from '../src/mediaTypes.js';
 
+const PREBID_VERSION = '$prebid.version$'
 const BIDDER_CODE = 'kargo';
-const HOST = 'https://krk.kargo.com';
+const HOST = 'https://krk2.kargo.com';
 const SYNC = 'https://crb.kargo.com/api/v1/initsyncrnd/{UUID}?seed={SEED}&idx={INDEX}&gdpr={GDPR}&gdpr_consent={GDPR_CONSENT}&us_privacy={US_PRIVACY}';
 const SYNC_COUNT = 5;
 const GVLID = 972;
 const SUPPORTED_MEDIA_TYPES = [BANNER, VIDEO];
 const storage = getStorageManager({gvlid: GVLID, bidderCode: BIDDER_CODE});
+const CURRENCY_KEY = 'currency'
+const US_DOLLAR_KEY = 'USD'
+const TDID_BID_KEY = 'userId.tdid'
+const SOCIAL_CANVAS_BID_KEY = 'params.socialCanvas'
+const SUA_BID_KEY = 'ortb2.device.sua'
+const SUA_ATTRIBUTES = ['browsers', 'platform', 'mobile', 'model', 'source']
+const MOBILE = 'mobile'
+const MODEL = 'model'
```

Comment:
This isn't used. How about consolidating these into groupings of different objects for easier referencing? Something like this make sense to me. i.e:
```
const CURRENCY = Object.freeze({
  US_DOLLAR: 'USD'
});

const REQUEST_KEYS = Object.freeze({
  CERBERUS: 'krg_crb',
  CURRENCY: 'currency',
  SOCIAL_CANVAS: 'params.socialCanvas',
  SUA: 'ortb2.device.sua',
  TDID: 'userId.tdid',
  US_DOLLAR: 'USD',
});

const SUA = Object.freeze({
  BROWSERS: 'browsers',
  MOBILE: 'mobile',
  MODEL: 'model',
  PLATFORM: 'platform',
  SOURCE: 'source',
});
const SUA_ATTRIBUTES = [
  SUA.BROWSERS,
  SUA.MOBILE,
  SUA.MODEL,
  SUA.SOURCE,
  SUA.PLATFORM,
];

const CERBERUS = Object.freeze({
  KEY: 'krg_crb',
  PAGE_VIEW_ID: 'pageViewId',
  PAGE_VIEW_TIMESTAMP: 'pageViewTimestamp',
  PAGE_VIEW_URL: 'pageViewUrl'
});
```